PowerShell/BuildingBlocks/SDKOperations/RetrieveEntities.ps1

```diff
@@ -0,0 +1,39 @@
+# Initialize the context
+.\Init.ps1
+
+# Here are 3 common ways to retrieve entities
+# 1. Retrieve all the customers in a workgroup and process them one by one
+# Use -RetrieveAll and piping with ForEach 
+Get-BT_Customer -Ticket $mspc.WorkgroupTicket -RetrieveAll | ForEach {
```

Comment:
You can't use a workgroupTicket to retrieve Customer entities, because it is Customer-scoped.
If you wan't to retrieve all customers under a given workgroup, you should use a regular ticket ($mspc.Ticket) and filter by -WorkgroupId.

This sample script is just to illustrate the retrieve operation, so I think we can simplify it and either:
1) Keep the version you had in the code review (i.e. just retrieve all customers, with $mspc.Ticket)
2) Use another entity (e.g. Endpoint, which would need a $customerTicket).

#1 might cause a lot of records to be returned, so we may want to go for #2

Reply:
OK.